### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MentalGear**
Clarity: Added the authors' original academic essay "Local-first software" in the readme to make it easier to understand how automerge emerged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ and merging**:
 
 ## Features and design principles
 
+Automerge is spawn from the [academic essay "Local-first software"](https://www.inkandswitch.com/local-first.html) that compares various approaches to automatic conflict-resolution and details the reasoning behind how automerge came to be.
```
Comment (aslakhellesoy) [inline README.md:46]: The [first commit](3a68032e284b107302a08aa7cf172b8c5816f3f4) in Automerge happened 2 years before that article was published...

Comment (MentalGear) [reply README.md:46]: Huh? It states "Ink & Switch has developed an open-source, JavaScript CRDT implementation called Automerge."

Anyways, one can rephrase the text, but the essay is still really helpful.

```suggestion
Automerge is featured in the [academic essay "Local-first software"](https://www.inkandswitch.com/local-first.html) that compares various approaches to automatic conflict-resolution and explains the workings behind why automerge is a preferable solution.
```

**Comment by pvh** [comment]
Yes, we started automerge, then known as tesseract, two years before the essay was published based on Martin's research. After an initial prototype, Martin took over development in a collaboration with the lab, and eventually, we spun automerge out into its own GitHub organization.

**Comment by MentalGear** [comment]
@Pvh Would you then be able to revise the suggested readme text linking to the essay?
No matter how phrased, the essay is essential information to have (several people already asked me about information that is in the essay, but they couldn't find in the readme)

**Comment by ept** [comment]
@MentalGear Thanks for suggesting this, it's a good idea to link to the local-first essay. I've rephrased it a bit and added a bit more detail.

**Pull request merged**